features/polyPinion/src/screens/navigation/AppNavigator.tsx

```diff
@@ -52,15 +52,16 @@ export default function AppNavigator() {
                     // Only change the language in case it needed. This will only happen the first
                     // time when opening the app, as then savedLanguage will be null.
                     if (savedLanguage === null) {
+                        // TODO: following line actually does nothing! there's no storage
```

Comment:
Not sure I understand this comment @JJ - you propose to not even call the `storeLanguage` here?
I wanted to avoid "touching" this part of code at a structural level, just marked some things on the way as TODOs to make our future lives a bit easier :P

Reply:
Well, it does nothing...